packages/flutter/lib/src/material/text_field.dart

```diff
@@ -424,9 +453,7 @@ class TextField extends StatefulWidget {
   final DragStartBehavior dragStartBehavior;
 
   /// {@macro flutter.rendering.editable.selectionEnabled}
-  bool get selectionEnabled {
-    return enableInteractiveSelection ?? !obscureText;
-  }
+  bool get selectionEnabled => enableInteractiveSelection ?? true;
```

Comment:
The docs say `enableInteractiveSelection` is true by default. What does it mean if it is null? (Are we just missing an assert in the constructor that this cannot be null?)

Reply:
it default to null, I have though about default it to true in constructor and just do ```bool get selectionEnabled => enableInteractiveSelection```

but that will require assert(enableInteractiveSelection!= null) which will change api behavior, previously we allow user passed in null.

Reply:
I think changing it to include the assert will be fine. That we don't have it is more like a bug...

Reply:
we dont have that because we default it based on obscureText. I will fix this, do you think this will be a breaking change?